extensions/wikia/AdEngine/js/DOMElementTweaker.js

```diff
@@ -54,18 +54,26 @@ define('ext.wikia.adEngine.domElementTweaker', [
 		}
 	}
 
-	function recursiveMoveStylesToInline(element) {
+	function cleanInlineStyles(element) {
+		if (isElement(element)) {
+			element.removeAttribute('style');
+			element.style.cssText = null;
```

Comment:
I'd write: `element.style.cssText = '';`
and do we really need to remove attribute and not just clearing `cssText`?